teku/src/main/java/tech/pegasys/teku/AbstractNode.java

```diff
@@ -40,6 +41,7 @@
 
 public abstract class AbstractNode implements Node {
   private static final Logger LOG = LogManager.getLogger();
+  private static final long intervalBetweenPublications = 60;
```

Comment:
generally `private static final` are named like a constant, all caps and underscores like `INTERVAL_BETWEEN_PUBLICATIONS`